internal/pkg/template/templates/environment/partials/environment-manager-role.yml

```diff
@@ -13,6 +13,9 @@ EnvironmentManagerRole:
         Principal:
           AWS: !Sub ${ToolsAccountPrincipalARN}
         Action: sts:AssumeRole
+{{- if .PermissionsBoundary }}
+    PermissionsBoundary: !Sub 'arn:${AWS::Partition}:iam::${AWS::AccountId}:policy/{{.PermissionsBoundary}}'
```

Comment:
Are we planning to update the other IAM roles in the env stack in a separate PR? For example: https://github.com/aws/copilot-cli/blob/4b45d6682865598b368819023f15185b6d572b93/internal/pkg/template/templates/environment/partials/custom-resources-role.yml#L2 

I think for that we will need to touch `env package` and `env deploy` as well.

Reply:
Yup!